node/modules/core.go

```diff
@@ -71,7 +71,7 @@ func MemoryConstraints() system.MemoryConstraints {
 // MemoryWatchdog starts the memory watchdog, applying the computed resource
 // constraints.
 func MemoryWatchdog(lr repo.LockedRepo, lc fx.Lifecycle, constraints system.MemoryConstraints) {
-	if os.Getenv(EnvWatchdogDisabled) == "1" {
+	if true || os.Getenv(EnvWatchdogDisabled) == "1" {
```

Comment:
Probably drop before merging

Reply:
Yep. Although I'd like to figure out _why_ this is an issue before releasing.